docker-app/qfieldcloud/core/tests/test_job.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from django.forms.models import ValidationError
+from qfieldcloud.authentication.models import AuthToken
+from qfieldcloud.core.models import Job, Person, Project
+from qfieldcloud.subscription.models import Subscription
+from rest_framework.test import APITransactionTestCase
+
+from .utils import setup_subscription_plans
+
+logging.disable(logging.CRITICAL)
+
+
+class QfcTestCase(APITransactionTestCase):
+    def setUp(self):
+        setup_subscription_plans()
+
+        # Create a user
+        self.user1 = Person.objects.create_user(username="user1", password="abc123")
+        self.token1 = AuthToken.objects.get_or_create(user=self.user1)[0]
+
+        # Create a user
+        self.user2 = Person.objects.create_user(username="user2", password="abc123")
+        self.token2 = AuthToken.objects.get_or_create(user=self.user2)[0]
+
+        # Create a user
+        self.user3 = Person.objects.create_user(username="user3", password="abc123")
+        self.token3 = AuthToken.objects.get_or_create(user=self.user3)[0]
+
+        # Create a project
+        self.project1 = Project.objects.create(
+            name="project1", owner=self.user1, description="desc", is_public=False
+        )
+
+    def test_create_job_succesfully(self):
+        job = Job.objects.create(
+            type=Job.Type.PACKAGE, project=self.project1, created_by=self.user1
+        )
+
+        self.assertEqual(job.status, Job.Status.PENDING)
+
+    def test_create_job_by_inactive_user(self):
+        # set_subscription(
+        #     self.user1, initial_subscription_status=Subscription.Status.INACTIVE_DRAFT
+        # )
+        self.user1.useraccount.active_subscription.status = (
+            Subscription.Status.INACTIVE_DRAFT
+        )
+        self.user1.useraccount.active_subscription.save()
+
```

Comment:
and maybe here, how to best setup so that the `useraccount.active_subscription.is_active id False` ?